extensions/slack/src/search.tsx

```diff
@@ -0,0 +1,65 @@
+import { Action, ActionPanel, Image, List } from "@raycast/api";
+
+import { CacheProvider, useChannels, useGroups, useUsers } from "./shared/client";
+import { openChannel, openChat } from "./shared/utils";
+
+export default function Command() {
+  return (
+    <CacheProvider>
+      <SlackList />
+    </CacheProvider>
+  );
+}
+
+function SlackList() {
+  const { data: users } = useUsers();
+  const { data: channels } = useChannels();
+  const { data: groups } = useGroups();
+
+  return (
+    <List isLoading={!users || !groups || !channels}>
+      <List.Section title="Users">
+        {users?.map(({ name, id, teamId, icon }) => (
```

Comment:
```suggestion
        {users?.map(({ name, username, id, teamId, icon }) => (
```

Reply:
Only a suggestion

Reply:
Hmm, I would prefer to use the name as it is right now

Reply:
Could still make sense to have a subtitle with the username then, maybe it's just me :)